test/controllers/income/estimatedPay/update/IncomeUpdateCalculatorControllerSpec.scala

```diff
@@ -83,30 +84,339 @@ class IncomeUpdateCalculatorControllerSpec
       .thenReturn(Future.successful(employer.name))
   }
 
-  val employment =
-    Employment(employer.name, Some("123"), new LocalDate("2016-05-26"), None, Nil, "", "", 1, None, false, false)
-  val taxCodeIncome = TaxCodeIncome(
-    EmploymentIncome,
-    Some(employer.id),
-    1111,
-    "employer",
-    "S1150L",
-    "employer",
-    OtherBasisOfOperation,
-    Live)
+  object UnifiedHarness {
+    def BuildTaxCodeIncomes(incomeCount: Int) = {
```

Comment:
This stuff is less harness related and more fixture like so this doesn't seem to be the right place for it